DrawerLayout/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "main": "../dist/src/components/DrawerLayout",
-  "react-native": "../src/components/DrawerLayout",
-  "browser": "../src/components/DrawerLayout",
-  "types": "../dist/src/components/DrawerLayout.d.ts"
+  "main": "../lib/module/components/DrawerLayout",
+  "react-native": "../lib/module/components/DrawerLayout",
```

Comment:
I like to specify the source file for `react-native` since metro already compiles these and it makes debugging easier

Reply:
Fixed in de7e490.